scripts/create_release.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import subprocess
+import sys
+from collections import OrderedDict
+
+script_dir = os.path.dirname(os.path.abspath(__file__))
+root_dir = os.path.dirname(script_dir)
+sys.path.append(root_dir)
+
+import emsdk  # noqa
+
+
+def version_to_list(version_string):
+  return [int(part) for part in version_string.split('.')]
+
+
+def main(args):
+  if subprocess.check_output(['git', 'status', '--porcelain'], cwd=root_dir).strip():
+    print('tree is not clean')
+    sys.exit(1)
+
+  release_info = emsdk.load_releases_info()
+  new_version = version_to_list(release_info['latest'])
+  new_version[-1] += 1
+  branch_name = 'version_%s' % '_'.join(str(part) for part in new_version)
+
+  # Create a new git branch
+  subprocess.check_call(['git', 'checkout', '-b', branch_name], cwd=root_dir)
+
+  new_version = '.'.join(str(part) for part in new_version)
+  new_hash = emsdk.get_emscripten_releases_tot()
```

Comment:
Doesn't have to be in the first commit, but we should add support for picking a particular revision as the release. Maybe this script could have 2 entry points, one that creates the CL in emscripten-releases to make the build, and another that starts from here and does the rest.